src/Sylius/Bundle/ApiBundle/Resources/config/routing/taxon.yml

```diff
@@ -1,11 +1,19 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
-sylius_admin_taxon:
+sylius_api_taxon:
```

Comment:
I thought about placing it in `src/Sylius/Bundle/ApiBundle/Resources/config/routing/taxon.yml` but ok :)